tests/loops/test_loops.py

```diff
@@ -344,7 +344,8 @@ def val_dataloader(self):
     trainer.fit_loop.load_state_dict(checkpoint)
 
     # `nbe_`: non-breaking epoch, as in, no exception will be raised. `be_`: breaking epoch
-    nbe_total_val_batch = stop_epoch * n_dataloaders * n_batches
+    # the fit-validation total batch progress is reset per epoch so it's not counted for the total value.
+    nbe_total_val_batch = 0  # stop_epoch * n_dataloaders * n_batches
```

Comment:
why to keep this comment?

Reply:
because it's the expected `nbe_total_val_batch` value if we didn't do the special behaviour implemented in this PR